src/test/java/internship/tests/NavigationPageTest.java

```diff
@@ -11,11 +11,10 @@
 
     @Test(testName = "Open Navigation page",
             description = "Open Navigation page and verify sections",
-            dataProvider = "sectionNavigation", dataProviderClass = DataProviders.class)
+            dataProvider = "sectionNavigation", dataProviderClass = DataProviders.class,
+            priority = 2)
```

Comment:
why you need this?

Reply:
It seems that I don't need priority for my tests. Fixed.